### DISCUSSION TITLE
Installation issue: no Mitsuba Import/Export options

### DISCUSSION THREAD

**Discussion by gioarma** [mitsuba-renderer/mitsuba-blender#78] [Q&A] [answered]
Hello,

I'm on a Windows machine and I'm having troubles making the plugin work. I think I have setup everything as indicated in the Wiki, but when I go to File -> Export, i see no Mitsuba .xml option, even after restarting Blender several times:

To setup the plugin I had to compile mitsuba and use a custom Mitsuba path because the plugin was not failing to automatically load the Mitsuba package:
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/59478987/393abb19-4463-4d58-a9e8-3f4e664498e3)

So, I compiled mitsuba, which correctly works, and manually specified the `mitsuba3/build` folder (is this correct?).
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/59478987/82b08563-1c54-4ee7-ae14-fb32504aa7bf)

Now I have no errors after restarting Blender, but I cannot see any Mitsuba export option, so I guess there is still something wrong. Can anyone tell me what I might have done wrong? Thank you!
![image](https://github.com/mitsuba-renderer/mitsuba-blender/assets/59478987/a24abc64-11b5-4de1-892c-7b9c2123fe61)

**Comment by bathal1** [comment]
Which version of blender are you using ?

**Reply by gioarma** [reply]
It's version `3.4.1`

**Comment by bathal1** [comment, marked as answer]
For blender 3.4, you will need to run blender from a command prompt, using the `--python-use-system-env` argument. More details are given in [this](https://github.com/mitsuba-renderer/mitsuba-blender/issues/61#issuecomment-1587980110) comment.

**Reply by gioarma** [reply]
Yes! running the following on the terminal solved the issue!
```python
"C:\Program Files\Blender Foundation\Blender 3.4\blender.exe" --python-use-system-env
```

Which Blender version should I use to avoid having to run it from the terminal?

**Reply by bathal1** [reply]
Versions of blender from 3.5 onwards use this flag by default. Unfortunately, this seems to have broken the addon. We're trying to resolve this issue but it currently simply doesn't work.

The best I can suggest for the moment if you want to avoid running it from the terminal is to modify the command that is run by your blender shortcut (by right-clicking on it and going to `Properties`) such that it includes this argument.
